### PR TITLE
IEP-870: Missing gdb executable for esp32c2 target

### PR DESCRIPTION
**Pull request by sigmaaa**
## Description

Fixes # ([IEP-870](https://jira.espressif.com:8443/browse/IEP-870))

## Type of change

Please delete options that are not relevant.
- Bug fix (non-breaking change which fixes an issue)

## How has this been tested?

Test 1:
- in the debug configuration select esp32c2 target -> select esp32c2 board

**Test Configuration**:
* ESP-IDF Version:
* OS (Windows,Linux and macOS):

## Dependent components impacted by this PR:

- Debug configuration

## Checklist
- [ ] PR Self Reviewed
- [ ] Applied Code formatting
- [ ] Added Documentation
- [ ] Added Unit Test
- [ ] Verified on all platforms - Windows,Linux and macOS

### CODE REVIEW

File: bundles/com.espressif.idf.core/src/com/espressif/idf/core/util/IDFUtil.java [hunk 1]
```diff
@@ -43,6 +45,11 @@
  */
 public class IDFUtil
 {
+
+	private IDFUtil()
+	{
+	}
```
Comment (kolipakakondal) [inline bundles/com.espressif.idf.core/src/com/espressif/idf/core/util/IDFUtil.java:51]: Thanks for adding this. It makes sense as all the methods are static we can avoid instantiation.

**Review by kolipakakondal — APPROVED** [review]
LGTM

**Comment by kolipakakondal** [comment]
@AndriiFilippov. ^^

**Comment by AndriiFilippov** [comment]
@sigmaaa @kolipakakondal hi ! 

Tested under: 
OS - Windows 10 / Linux 
ESP-IDF: master / v5.0 

LGTM! 

image

**Review by AndriiFilippov — APPROVED** [review]

**Pull request merged**